docs/kyma/04-03-cluster-installation.md

```diff
@@ -28,14 +22,22 @@ Choose your cloud provider and get started:
 Install Kyma on a [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) (GKE) cluster.
 
 ## Prerequisites
+
 - [Google Cloud Platform](https://console.cloud.google.com/) (GCP) project with Kubernetes Engine API enabled
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.12.0
 - [gcloud](https://cloud.google.com/sdk/gcloud/)
 - [wget](https://www.gnu.org/software/wget/)
 
-## Installation process
+## Choose the release to install
+
+1. Go to [this](https://github.com/kyma-project/kyma/releases/) page and choose the release you want to install.
+
+2. Export the release version as an environment variable. Run:
+    ```
+    export KYMA_VERSION={KYMA_RELEASE_VERSION}
+    ```
 
-### Prepare the GKE cluster
+## Prepare the GKE cluster
```

Comment:
Can you use ` --cluster-version "1.12" instead of ` --cluster-version "1.12.5"` and would that be better in any way?

Reply:
We can use that, according to GKE docs: `1.X: Specifies the highest valid patch+gke.N patch release in the 1.X minor version`

Reply:
I've changed that.